azure-steps-windows.yml

```diff
@@ -28,26 +28,33 @@ steps:
     mkdir  C:/opt/openblas/openblas_dll
     mkdir  C:/opt/32/lib/pkgconfig
     mkdir  C:/opt/64/lib/pkgconfig
-    # TBD: support 32 bit testing
     $target=$(python -c "import tools.openblas_support as obs; plat=obs.get_plat(); ilp64=obs.get_ilp64(); target=f'openblas_{plat}.zip'; obs.download_openblas(target, plat, ilp64);print(target)")
```

Comment:
Oh, this is ugly :)

Reply:
Yes, this stuff is awful. I'm really looking forward to the day we have OpenBLAS in a wheel and get rid of this.